src/fparser/__init__.py

```diff
@@ -65,11 +65,23 @@
 # First version by: Pearu Peterson <pearu@cens.ioc.ee>
 # First created: Oct 2006
 
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except ModuleNotFoundError:
+    from importlib_metadata import version, PackageNotFoundError
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    from setuptools_scm import get_version
+
+    __version__ = get_version(root="..", relative_to=__file__)
```

Comment:
Is this path correct? I *think* `..` will take us to the `src` directory rather than the root of the project?

Reply:
Good point, this probably wants to be `../..`